website/docs/core-concepts/stacks/yaml-functions/env.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "!env"
+sidebar_position: 5
+sidebar_label: "!env"
+description: Retrieve the value of an environment variable named by the key, and assign the result to the sections in Atmos stack manifests
+---
+
+import Intro from '@site/src/components/Intro'
+
+<Intro>
+    The `!env` Atmos YAML function is used to retrieve the value of an environment variable named by the key,
+    and assign the result to the sections in Atmos stack manifests.
+</Intro>
+
+## Usage
+
+```yaml
+vars:
+  api_key: !env API_KEY
+
+settings:
+  provisioned_by_user: !env USER
+```
+
+The value of the retrieved environment variable can be a simple type (string, number, or boolean), in which case Atmos
+assigns it without modification.
+
+If the value is a complex type (list, map or object), it must be returned as a JSON-encoded string.
+After receiving the JSON-encoded string, Atmos automatically decodes it into the corresponding YAML complex type.
+
+:::tip
+You can use [Atmos Stack Manifest Templating](/core-concepts/stacks/templates) in the `!env` YAML function.
+Atmos processes the templates first, and then executes the `!env` function, allowing you to provide the parameters to
```

Comment:
I don't know that evaluating the env is a good/safe idea. Because that can then contain expressions that then execute commands.

Reply:
Do we need that behavior?

Reply:
If we support that, we might want to move that to a different function, like, `!env.eval`, and leave !env simple for simple scalars.